docs/asciidoc/faq.adoc

```diff
@@ -4,9 +4,84 @@
 This section covers the following content:
 
 * <<faq.proxy-connect-method>>
+* <<faq.logging-prefix>>
+* <<faq.logging-correlation>>
 
 [[faq.proxy-connect-method]]
 == Connection to the proxy cannot be established
 include::proxy.adoc[tag=proxy-connect-method]
 
+[[faq.logging-prefix]]
+== What is it meaning of the information that is prepended to every log record?
+Reactor Netty adds information for the connection at the beginning of every log record (when this is possible).
+There is a slight difference in the details for the connection when you use `TCP`, `UDP`, `HTTP/1.1` or `HTTP/2`.
 
+=== TCP and UDP
+In case of `TCP` and `UDP`, the following is added at the beginning of every log record: the id of the underlying connection, local and remote addresses.
+
+====
+[source,text,indent=0]
+----
+Examples
+[a1566d55, L:/[0:0:0:0:0:0:0:1]:53446 - R:/[0:0:0:0:0:0:0:1]:53444]
+[a1566d55, L:/[0:0:0:0:0:0:0:1]:53446 ! R:/[0:0:0:0:0:0:0:1]:53444]
+
+Format
+[<CONNECTION_ID>, L:<LOCAL_ADDRESS> <CONNECTION_OPENED_CLOSED> R:<REMOTE_ADDRESS>]
+<CONNECTION_ID>: a1566d55
+<LOCAL_ADDRESS>: [0:0:0:0:0:0:0:1]:53446
+<REMOTE_ADDRESS>: [0:0:0:0:0:0:0:1]:53444
+<CONNECTION_OPENED_CLOSED>: - (connection opened)
```

Comment:
nitpick but I'd put that in the same order as it appears in the log line, ie. before REMOTE_ADDRESS line 33.
same for the other "format" sections (HTTP1, HTTP2...) below

Reply:
I fixed that, please check now